tests/duty.data.ts

```diff
@@ -0,0 +1,25 @@
+import { ObjectId } from 'mongodb';
+import { DutySchema } from '../services/schemas/duty.zschema';
+
+const createTestDuties = (amount: number) => {
+  return Array.from({ length: amount }, () => {
+    return DutySchema.parse({
+      _id: new ObjectId(),
+      name: Math.random().toString(36).substring(3, 9),
```

Comment:
There are [libraries](https://www.npmjs.com/package/@faker-js/faker) for this thing

Reply:
resolved